trongtruong/sale_training/wizard/add_orders.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, fields, api
+
+
+class AddTagToOrders(models.TransientModel):
+    _name = 'add.orders'
+
+    order = fields.Many2many(comodel_name="sale.order", string="Order",
+                             domain=[('state', 'not in', ('draft', 'sent', 'cancel'))])
+
+    @api.multi
+    def add_orders(self):
+        if self.order:
+            for i in self.order:
```

Comment:
Sửa lại thành for order in self.order_ids
Không nhất thiết phải check if self.order_ids trong trường hợp này.